.github/actions/run-airbyte-ci/action.yml

```diff
@@ -99,64 +102,32 @@ runs:
           exit 78
         fi
 
-    - name: Get changed files
-      uses: tj-actions/changed-files@v39
-      id: changes
-      with:
-        files_yaml: |
-          pipelines:
-            - 'airbyte-ci/connectors/pipelines/**'
-
-    - name: Determine how Airbyte CI should be installed
+    - name: Set Docker registry mirror
+      id: set-docker-registry-mirror
+      if: inputs.docker_registry_mirror_url
       shell: bash
-      id: determine-install-mode
       run: |
-        if [[ "${{ github.ref }}" != "refs/heads/master" ]] && [[ "${{ steps.changes.outputs.pipelines_any_changed }}" == "true" ]]; then
-          echo "Making changes to Airbyte CI on a non-master branch. Airbyte-CI will be installed from source."
-          echo "install-mode=dev" >> $GITHUB_OUTPUT
-        else
-          echo "install-mode=production" >> $GITHUB_OUTPUT
-        fi
+        sudo systemctl stop docker
+        echo '{"registry-mirrors": ["https://${{inputs.docker_registry_mirror_url}}"]}' | sudo tee /etc/docker/daemon.json
+        sudo systemctl start docker
```

Comment:
Not sure how long this takes/if it is an issue at all. 
 
 But looks like you can force the docker daemon to reload its configuration without restarting: 
 > Restart the Docker daemon. On Linux, you can avoid a restart (and avoid any downtime for your containers) by reloading the Docker daemon. If you use systemd, then use the command systemctl reload docker. Otherwise, send a SIGHUP signal to the dockerd process.
 
 (from [here](https://docs.docker.com/config/containers/live-restore/))

Reply:
Thanks for the suggestion, I did not notice any related slowness, but if I do I'll consider it!